src/pages/Home/components/Home.test.js

```diff
@@ -0,0 +1,22 @@
+import { render, screen } from "@testing-library/react";
+
+import HomePage from '../Home.page';
+describe('ListItem', () => {
+    test('Should create a container for home page', () => {
+        const item = render(HomePage())
+        const homeContent = item.container.querySelector('.homepage')
+        expect(homeContent).toBeDefined()
+    })
+
+    test('Should contain a title', () => {
+        render(HomePage())
+        const homeContent = screen.queryByText(/Welcome to the challenge/i)
```

Comment:
You should use `query*` only if you are testing a non existence component. In this case, you can use `getByText`. See: https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-query-variants-for-anything-except-checking-for-non-existence for more common mistakes while creating tests